src/main/java/frc/bionic/Conversion.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Team 4909, Bionics
+ * Billerica Memorial High School
+ *
+ * Copyright:
+ *   2021 Bionics
+ *
+ * License:
+ *   MIT: https://opensource.org/licenses/MIT
+ *   See the LICENSE file in the project's top-level directory for details.
+ */
+
+package frc.bionic;
+
+public class Conversion
+{
+  /**
+   * Convert inches to meters
+   *
+   * @param inches
+   *   Value to convert
+   *
+   * @return
+   *   Converted value
+   */
+  static public double inchesToMeters(double inches)
+  {
+    return inches * 0.0254;
+  }
+
+  /**
+   * Convert meters to inches
+   *
+   * @param meters
+   *   Value to convert
+   *
+   * @return
+   *   Converted value
+   */
+  static public double metersToInches(double meters)
+  {
+    return meters * 39.3701;
+  }
+
+  /**
+   * Convert wheel revolutions to meters traveled
+   *
+   * @param revers
+   *   Number of revolutions traveled
+   *
+   * @param wheelDiameterMeters
+   *   The diameter of the wheel
+   *
+   * @return
+   *   Equivalent number of meters traveled
+   */
+  static public double wheelRevsToMetersTraveled(double revs, double wheelDiameterMeters)
+  {
+    double          circumferenceMeters = Math.PI * wheelDiameterMeters;
```

Comment:
why so much space?

Reply:
I like to line up all of my variables, so I have a keyboard shortcut that indents them to a consistent place.